scico/functional/_indicator.py

```diff
@@ -50,17 +48,17 @@ def prox(
         self, v: Union[JaxArray, BlockArray], lam: float = 1.0, **kwargs
     ) -> Union[JaxArray, BlockArray]:
         r"""Evaluate the scaled proximal operator of the indicator over
-            the non-negative orthant, :math:`I_{>= 0} `,:
+        the non-negative orthant, :math:`I_{>= 0}`
```

Comment:
I think there needs to be some connector here, because the equation on the following lines is part of this sentence (and is an independent clause)

Reply:
Added a comma.